src/services/token.services.js

```diff
@@ -0,0 +1,30 @@
+const { Token } = require('../models/token');
+
+const getToken = async (userId) => {
+  const token = await Token.findOne({ where: { userId } });
+
+  return token;
+};
+
+const saveToken = async (userId, refreshToken) => {
+  const token = await getToken(userId);
+
+  if (!token) {
+    return Token.create({ userId, refreshToken });
+  }
+
+  token.refreshToken = refreshToken;
+  token.save();
```

Comment:
Consider using `await token.save();` to ensure the save operation completes before the function returns. This will prevent any potential issues with asynchronous operations.